recipe/meta.yaml

```diff
@@ -1,35 +0,0 @@
-{% set version = "1.12.1" %}
-
-package:
-  name: ninja
-  version: {{ version }}
-
-source:
-  url: https://github.com/ninja-build/ninja/archive/v{{ version }}.tar.gz
-  sha256: 821bdff48a3f683bc4bb3b6f0b5fe7b2d647cf65d52aeb63328c91a6c6df285a
-
-build:
-  number: 0
-
-requirements:
-  build:
-    - {{ compiler("cxx") }}
-    - {{ stdlib("c") }}
-    - python *
-
-test:
-  commands:
-    - ninja --version
```

Comment:
This test made sure the ninja command at least ran as far as producing a version, while the new one just checks to see if there's a binary present?

Reply:
Makes sense, I added it back